enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/SecuritySettings.java

```diff
@@ -130,4 +132,12 @@
                   "E.g. group1=role1;group2=role2;group3=role3,role4,role5" )
     public static Setting<String> ldap_authorization_group_to_role_mapping =
             setting( "dbms.security.realms.ldap.authorization.group_to_role_mapping", STRING, NO_DEFAULT );
+
+    @Description( "The time to live (TTL) for cached authentication and authorization info." )
+    public static Setting<Long> auth_caching_ttl =
```

Comment:
`auth_cache_ttl` is more consistent with the next setting (and better to change this than have auth_caching_max_capacity which sounds weird)